Sitenotice.php

```diff
@@ -10,7 +10,7 @@
 // and don't comment it out
 $wgMajorSiteNoticeID = 55;
 
-// if ( !$wmgSiteNoticeOptOut ) {
+/* // if ( !$wmgSiteNoticeOptOut ) {
```

Comment:
Can you please remove the `//` from here? While not really required I do think it simply looks better like that. Thanks!

Reply:
@Universal-Omega Can you not make the change for me? For such a minor change, it seems a bit bureaucratic to have to make the requestor do it.

Reply:
sure, I can I just typically don't like modifying others' PRs, but I can do that if you want.